book/src/leader-rotation.md

```diff
@@ -3,17 +3,76 @@
 At any given moment, a cluster expects only one fullnode to produce ledger
 entries. By having only one leader at a time, all validators are able to replay
 identical copies of the ledger. The drawback of only one leader at a time,
-however, is that a malicious leader is cabable of censoring votes and
+however, is that a malicious leader is capable of censoring votes and
 transactions. Since censoring cannot be distinguished from the network dropping
 packets, the cluster cannot simply elect a single node to hold the leader role
-indefinitely. Instead, the cluster minimizes the influence of a malcioius
+indefinitely. Instead, the cluster minimizes the influence of a malicious
 leader by rotating which node takes the lead.
 
 Each validator selects the expected leader using the same algorithm, described
 below. When the validator receives a new signed ledger entry, it can be certain
 that entry was produced by the expected leader.
 
-## Leader Schedule Generation
+## Leader Schedule Rotation
+
+A block's slot is valid if it is generated by the leader computed from a Leader
+Schedule in the block that crossed the epoch slot boundary at least 1 epoch ago.
+
+Leader schedule is generated every epoch when a fork crosses the epoch boundary.
+In practice the schedule only needs to be generated when the root fork crosses
+the epoch boundary.
+
+The LS is active at the next epoch.  The epoch should be long enough such that
+by the time next epoch is reached, the LS is based on a fork that has been
+committed as root.
+
+Without an epoch-scale partition, the network will work as follows:
+
+1. The root fork is updated as a validator votes for new forks.
+
+2. When the root fork slot height crosses the epoch boundary, the leader
+schedule is updated for the next epoch.
+
+3. Since the schedule is for the next epoch, any new stakes committed to the
```

Comment:
This doesn't need to be part of the list. It explains a consequence of how often the schedule is generated, which is mentioned earlier in this document. Consider moving this sentence up there.